From 29960f2fce7ab42ed0a1bf8ce96499fdb6eedf3c Mon Sep 17 00:00:00 2001
From: Simon Busch <simon.busch@lge.com>
Date: Tue, 25 Mar 2014 15:28:54 +0100
Subject: [PATCH] Don't use gobject introspection for bindings

In our environment we're not providing the glib python bindings through
introspection but just pygobject so we need to use the right import.

Signed-off-by: Simon Busch <simon.busch@lge.com>
---
 dbusmock/__main__.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dbusmock/__main__.py b/dbusmock/__main__.py
index 44b63fb..bfd089c 100644
--- a/dbusmock/__main__.py
+++ b/dbusmock/__main__.py
@@ -51,7 +51,7 @@ def parse_args():
 
 if __name__ == '__main__':
     import dbus.mainloop.glib
-    from gi.repository import GLib
+    import glib as GLib
 
     args = parse_args()
     dbus.mainloop.glib.DBusGMainLoop(set_as_default=True)
-- 
1.7.9.5

